3-extra/1-count-words.js

```diff
@@ -25,12 +25,19 @@
 
 function countWords(string) {
   const wordCount = {};
-
+  if (string === "") {
+    return wordCount;
+  } else {
   // write code here
-
+  string.split(" ").forEach((word) => {
+    
+    // const numberOfOccurrences = string.match(new RegExp(word, "g")).length
+    if (wordCount[word]) {
+      wordCount[word] = wordCount[word] +1;
+    } else wordCount[word] = 1;
+  })}
   return wordCount;
 }
-
```

Comment:
I have used a simple comparison to compare and count words inside a forEach method